src/select/parts/styles.scss

```diff
@@ -121,3 +121,33 @@ $checkbox-size: awsui.$size-control;
     color: awsui.$color-text-disabled;
   }
 }
+
+.inline-label-trigger-wrapper {
+  margin-block-start: -7px;
+}
+
+.inline-label-wrapper {
+  margin-block-start: calc(awsui.$space-scaled-xs * -1);
+}
+
+.inline-label {
+  box-sizing: border-box;
+  display: inline-block;
+  color: awsui.$color-text-form-label;
+  font-weight: awsui.$font-display-label-weight;
+  font-size: 12px;
+  line-height: 14px;
```

Comment:
How would this work with awsui.$line-height-body-s? (that's 16px though)

Reply:
I think this needs to be line-height 14px to satisfy the design requirements